src/Illuminate/Support/helpers.php

```diff
@@ -1,11 +1,12 @@
 <?php
 
-use Illuminate\Contracts\Support\Htmlable;
 use Illuminate\Support\Arr;
-use Illuminate\Support\Collection;
 use Illuminate\Support\Env;
-use Illuminate\Support\HigherOrderTapProxy;
+use Illuminate\Support\Str;
 use Illuminate\Support\Optional;
+use Illuminate\Support\Collection;
```

Comment:
I now noticed my editor ordered these classes by size. I can revert it if required.